runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4288,8 +4294,10 @@ TR::CompilationInfoPerThreadRemote::processEntry(TR_MethodToBeCompiled &entry, J
       J9Method *methodsOfClass = std::get<1>(classInfoTuple);
 
       // Build my entry
-      if (!(optPlan = TR_OptimizationPlan::alloc(optLevel)))
+      if (!(optPlan = TR_OptimizationPlan::alloc(clientOptPlan.getOptLevel())))
          throw std::bad_alloc();
+      optPlan->clone(&clientOptPlan);
```

Comment:
We may have to change `clone` in the mainline to copy the GPU fields as well.

Reply:
will create a separate PR for mainline